actions/dynamicdns

```diff
@@ -111,6 +112,21 @@ doGetOpt()
             ;;
         esac
     done
+    if [ $stdin -eq 1 ];then
+        # try to use with timeout and hidden input first
+        read -s -p "enter password and hit [ENTER]: " -t 10 password
+        RET=$?
```

Comment:
Bash is stripping away any spaces at the beginning and ending of the password.  For a password, this should not happen.